### DISCUSSION TITLE
Make giscus use the appropriate theme

### DISCUSSION THREAD

**Discussion by pulkitkrishna00** [thuliteio/doks#1019] [Q&A]
How to make giscus use the appropriate dark or light theme? It works if I switch theme from my system settings but it does not work if I change the theme from button on the website.

I found some discussion here: <https://github.com/giscus/giscus/issues/336>. How to apply it to doks?

**Comment by h-enk** [comment]
Doks 0.5 uses `<html lang="en-us" data-dark-mode>`. Doks 1.0 will use the [new Bootstrap way](https://getbootstrap.com/docs/5.3/customize/color-modes/#dark-mode): `<html lang="en-us" data-bs-theme="dark">`.

I'm not familiar with [giscus](https://github.com/giscus/giscus) (yet, it looks interesting), so I don't know how this translates.
